gettsim/transfers/rente.py

```diff
@@ -295,4 +413,361 @@ def ges_rente_regelaltersgrenze(geburtsjahr: int, ges_rente_params: dict) -> flo
             "intercepts_at_lower_thresholds"
         ],
     )
+
+    return out
+
+
+def ges_rente_frauen_altersgrenze(
+    geburtsjahr: int,
+    geburtsmonat: int,
+    ges_rente_params: dict,
+) -> float:
+    """Calculates the age, at which a women is eligible to claim the full pension. This
+        pension scheme allows for early retirement with deductions. Hence this
+        threshold is needed as reference for calculating the zugangsfaktor.
+
+
+    Parameters
+    ----------
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    geburtsmonat
+        See basic input variable :ref:`geburtsmonat <geburtsmonat>`.
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+
+    Returns
+    -------
+    returns the lowest full retirement age for women.
+
+    """
+    if geburtsjahr < 1945:
+        x_wom = geburtsjahr + (geburtsmonat - 1) / 12
+    else:
+        x_wom = geburtsjahr
+
+    out = piecewise_polynomial(
+        x=x_wom,
+        thresholds=ges_rente_params["altersrente_für_frauen"]["thresholds"],
+        rates=ges_rente_params["altersrente_für_frauen"]["rates"],
+        intercepts_at_lower_thresholds=ges_rente_params["altersrente_für_frauen"][
+            "intercepts_at_lower_thresholds"
+        ],
+    )
+
+    return out
+
+
+def _ges_rente_langjährig_altersgrenze(
+    geburtsjahr: int,
+    geburtsmonat: int,
+    ges_rente_params: dict,
+) -> float:
+    """Calculates the age, at which a long term insured person (at least 35 years )is
+        eligible to claim the full pension.
+
+    Parameters
+    ----------
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    geburtsmonat
+        See basic input variable :ref:`geburtsmonat <geburtsmonat>`.
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+    ges_rente_vorraus_besond_lang
+        See:func:`ges_rente_vorraus_besond_lang`.
+
+    Returns
+    -------
+    Full retirement age (without deductions) for long term insured.
+    """
+
+    if geburtsjahr < 1951:
+        x_long = geburtsjahr + (geburtsmonat - 1) / 12
```

Comment:
Just `x` ? Not sure what the `long` is supposed to mean.

Reply:
changed